test/unit/runnable.spec.js

```diff
@@ -142,9 +142,24 @@ describe('Runnable(title, fn)', function () {
           });
 
           test.run(function (err) {
-            calls.should.equal(1);
-            test.duration.should.be.type('number');
-            done(err);
+            if (err) {
+              done(err);
+              return;
+            }
+
+            try {
```

Comment:
this test was written in a way that assertions would still be made even if the callback received an error.